packages/framework-core/src/services/graphql/graphql-type-informer.ts

```diff
@@ -41,25 +44,56 @@ export class GraphQLTypeInformer {
   private metadataPropertiesToGraphQLFields(properties: Array<PropertyMetadata>): GraphQLFieldConfigMap<any, any> {
     const fields: GraphQLFieldConfigMap<any, any> = {}
     for (const prop of properties) {
-      fields[prop.name] = { type: this.getGraphQLTypeFor(prop.type) }
+      if (prop.typeInfo.type === Array) {
+        prop.typeInfo.parameters.forEach((param) => {
+          const graphQLPropType = this.getGraphQLTypeFor(param.type)
+
+          if (!this.canFilter(graphQLPropType)) {
+            const properties = getPropertiesMetadata(param.type)
+            this.generateGraphQLTypeFromMetadata({ class: param.type, properties })
+          }
+
+          fields[prop.name] = {
+            type: GraphQLList(graphQLPropType),
+          }
+        })
+      } else {
+        if (!prop.typeInfo.type.prototype) {
```

Comment:
I loved you included support for enums! But my question is Why this condition means the type is an enum?

Reply:
Because it's the only case the transformer returns an Enum. There is no other way of knowing if it's an enum, but this will be refactored later once we improve the transformer.